### PR TITLE
Refactor messy parts

### PR DESCRIPTION
**Pull request by dziemba**

### CODE REVIEW

File: lib/soapy_cake/admin_addedit.rb [hunk 1]
```diff
@@ -2,60 +2,8 @@ module SoapyCake
...
-
     def add_offer(opts = {})
-      require_params(opts, %i(
-        hidden offer_status_id offer_type_id currency_id ssl click_cookie_days
-        impression_cookie_days redirect_404 enable_view_thru_conversions
-        click_trumps_impression disable_click_deduplication last_touch
-        enable_transaction_id_deduplication postbacks_only fire_global_pixel
-        fire_pixel_on_nonpaid_conversions offer_link thankyou_link from_lines
-        subject_lines))
+      require_params(opts, Const::REQUIRED_NEW_OFFER_PARAMS)
```
Comment (hammerhead) [inline lib/soapy_cake/admin_addedit.rb]: I'd prefer to keep the `REQUIRED_NEW_OFFER_PARAMS` at the place where it semantically belongs to (the `AdminAddedit` class), rather then just stuffing everything in the generic `Const` class.

Comment (dziemba) [inline lib/soapy_cake/admin_addedit.rb]: Okay, I see your point. What about making a RequiredParameters class? Or a YAML file?

Comment (hammerhead) [inline lib/soapy_cake/admin_addedit.rb]: It's only used once in the whole project, so why not just keep it where it's used (`AdminAddedit`)?

**Comment by hammerhead** [comment]
:whale:

**Pull request merged**